01_RampUp/Examen/solucion_joseangel_diaz/ejercicio3.ps1

```diff
@@ -0,0 +1,18 @@
+#Mostrar todos los ficheros .DLL de la carpeta C:\Windows\System32 que sean anteriores 
+#a 2022 con un tamaño entre 10 y 50 kbytes.
+
+#Para mostar los archivos de la carpeta C:\Windows\System32 utilizo el siguiente comando:
+
+#Con -include *.DLL lo que hacemos es filtrar los archivos DLL y utilizamos | para concatenar los comandos.
+#Usamos LaswriteTimeyear junto con -le(menos o igual) 2021 para buscar los archivos anteriores al 22.
+
+#Delante de los 1000bytes he usado -ge para que sean mayor o igual que la cantidad y delante de 5000 -le para 
+#que sea menor o igual.
+
+#10kbytes = 1000bytes
+#50kbytes = 5000bytes.
+
+Get-ChildItem -Path "C:\Windows\System32" -Recurse -Include *.DLL |  Where-Object -FilterScript{
+    ($_.LastWriteTime.Year -le 2021) -and ($_.Length -ge 1000) -and ($_.Length -le 5000)}
```

Comment:
Solo dos cosas, en lugar de -Recurse -Include, prueba solo con -Filter. 
Y luego pide 10Kbytes y 50K bytes, lo que has puesto es 1Kbyte y 5KByte, te ha faltado un 0. 

Get-ChildItem -Path "C:\Windows\System32" -Filter *.DLL |  Where-Object -FilterScript{
    ($_.LastWriteTime.Year -lt 2022) -and ($_.Length -ge 10000) -and ($_.Length -le 50000)}